### DISCUSSION TITLE
Upload size on pocketbase

### DISCUSSION THREAD

**Discussion by Salimdevv1** [pocketbase/pocketbase#5351] [Q&A] [answered]
Hello everyone, im trying to upload an audio file of 300mb on pocketbase manually on the pb dashboard directly but i cannot idk why it appears an error 

data.error | Failed to load the submitted data due to invalid formatting.
-- | --
data.details | multipart: NextPart: EOF

im hosting pocketbase on coolify and here is the compose file 

services:
  pocketbase:
    image: 'ghcr.io/coollabsio/pocketbase:latest'
    environment:
      - SERVICE_FQDN_POCKETBASE_8080
      - PB_MAX_UPLOAD_SIZE=500MB
    volumes:
      - 'pocketbase-data:/app/pb_data'
      - 'pocketbase-hooks:/app/pb_hooks'

is there any solution  ?

**Comment by ganigeorgiev** [comment]
If PocketBase is behind a reverse proxy (eg. nginx) then most likely it is because of misconfiguration of some of the request limits - max allowed body size, read timeout, etc.

I see in the above that you have `PB_MAX_UPLOAD_SIZE` environment variable but this variable doesn't have any special meaning out of the box in PocketBase so it will depend on your image and Coolify setup whether the specified size will be acknowledged or not.

**Reply by Salimdevv1** [reply]
Noted , do you know how can i change the configuration on coolify? otherwise thank you for your response

**Reply by ganigeorgiev** [reply, marked as answer]
No sorry, I don't have experience with Coolify. You'll have better chance asking in their [discord channel](https://coolify.io/discord).

**Reply by Salimdevv1** [reply]
okay thank you
